http-client/src/test/java/io/airlift/http/client/AbstractHttpClientTest.java

```diff
@@ -85,6 +87,7 @@
     private String scheme = "http";
     private String host = "127.0.0.1";
     protected String keystore;
+    protected final Map<Integer, Integer> statusCounts = new ConcurrentHashMap<>();
```

Comment:
`Multiset<Integer>` seems better, since the intent is to have a count of the items